src/Window.cpp

```diff
@@ -910,7 +972,7 @@ void Window::removeMouseFromMaze() {
     m_commandBuffer.clear();
 
     // Reset movement state
-    m_startingLocation = {0, 0};
+    m_startingLocation = {1, 1};
```

Comment:
Let's leave this as `{0, 0}` since arrays start at zero. (It's arbitrary, but I don't see a good justification for changing it.)